### PR TITLE
[RFR] Add installer & check methods

### PR DESCRIPTION
**Pull request by floo51**
- [x] Add methods
- [x] Check methods
- [x] Remove methods

### CODE REVIEW

File: docs/js/installer/github.js [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+    const headers = {
+        Authorization: `token ${user.token}`,
+    };
+
+    // Response if user is a collaborator
+    // Status: 204 No Content
+    // Response if user is not a collaborator
+    // Status: 404 Not Found
+
+    return fetch(`${githubUrl}/repos/${user.name}/${repository.name}/collaborators/${sedyUsername}`, { headers})
```
Comment (Kmaschta) [inline docs/js/installer/github.js]: s/headers}/headers }

File: docs/js/installer/github.js [hunk 2]
```diff
@@ -0,0 +1,143 @@
+/* global fetch */
+const githubUrl = 'https://api.github.com';
```
Comment (alexisjanvier) [inline docs/js/installer/github.js]: I added the `config` module in #45. Maybe could we add those const in the `config/default.js` ?

File: docs/js/installer/github.js (hunk 1 above)
Comment (alexisjanvier) [inline docs/js/installer/github.js]: s/headers}/headers }/

File: docs/config/default.js [hunk 3]
```diff
@@ -1,9 +1,12 @@
 export default {
     appBaseUrl: 'http://localhost:8080',
     githubRedirection: '/setup.html',
-    githubAppId: 'xxx',
+    githubAppId: 'ea3f3d378a84370344bb',
```
Comment (Kmaschta) [inline docs/config/default.js]: shame

**Pull request merged**